src/app/participant-list/participant-list.component.ts

```diff
@@ -533,6 +538,21 @@ export class ParticipantListComponent implements OnInit {
     );
   }
 
+  private addDynamicFieldDefaultColumns(defaultColumn: any) {
+    outer: for (let sourceColumnGroup of Object.keys(this.sourceColumns)) {
```

Comment:
`Object.entries(this.sourceColumns)` would give you (key, value) and in below instead of `this.sourceColumns[sourceColumnGroup]` you would use `value` directly.